### ISSUE TITLE
vscjava.vscode-java-dependency does not have jars included

### ISSUE THREAD

**Issue by akosyakov** [open-vsx/publish-extensions#124]
```
Contains: Cannot extract bundle symbolicName or version /tmp/vscode-extensions/vscjava.vscode-java-dependency@0.9.0/extension/server/com.microsoft.jdtls.ext.core-0.9.0.jar
java.nio.file.NoSuchFileException: /tmp/vscode-extensions/vscjava.vscode-java-dependency@0.9.0/extension/server/com.microsoft.jdtls.ext.core-0.9.0.jar
```

**Comment by jankeromnes** [comment]
Notes: We used to auto-publish `vscjava.vscode-java-dependency` from this repository, but we've removed it when Microsoft claimed the `vscjava` namespace: https://github.com/open-vsx/publish-extensions/commit/784a105cbad5c15fa41b797cb6a30c48bf1550e4

However, no updated version of this extension has been published by Microsoft:
- https://open-vsx.org/extension/vscjava/vscode-java-dependency is still at our (old) `0.9.0` version
- while https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-dependency is already at `0.12.0`

If Microsoft has no interest in publishing `vscjava.vscode-java-dependency` to OpenVSX, I think we should resume doing it ourselves here.

Also, we need to figure out the correct `prepublish` step to run in order to include the missing JARs in our package. Their README.md seems to suggest `gulp build_server`.
